bundles/org.openhab.binding.wizlighting/.project

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think the file found accidently a way into your binding?

Reply:
Removed